recipes/capnproto/build.sh

```diff
@@ -5,7 +5,8 @@ cd build
 
 CMAKE_CXX_FLAGS="-fPIC"
 if [ "$(uname)" = "Darwin" ]; then
-    CMAKE_CXX_FLAGS="$CMAKE_CXX_FLAGS -stdlib=libc++"
+    # "-stdlib=libc++ -mmacosx-version-min=10.7" are required to enable C++11 features
+    CMAKE_CXX_FLAGS="$CMAKE_CXX_FLAGS -stdlib=libc++ -mmacosx-version-min=10.7"
```

Comment:
That one is added by Cap'n Proto itself (in fact, they set `-std=gnu+0x` to workaround something, so I prefer not to touch this).